src/crimson/os/alienstore/alien_store.cc

```diff
@@ -465,6 +465,31 @@ auto AlienStore::omap_get_values(CollectionRef ch,
   });
 }
 
+auto AlienStore::omap_iterate(CollectionRef ch,
+                              const ghobject_t &oid,
+                              const ObjectStore::omap_iter_seek_t &start_from,
+                              std::function<ObjectStore::omap_iter_ret_t(std::string_view, std::string_view)> &f,
+                              uint32_t op_flags)
+  -> read_errorator::future<ObjectStore::omap_iter_ret_t>
+{
+  logger().debug("{} with_start", __func__);
```

Comment:
Ultimately, we should use the new logging macros, see crimson/common/log.h or other users of `DEBUG` and `DEBUGDPP`. We could keep this out of scope and audit all other instances in alien_store in a separate PR.

Reply:
will switch logger.debug to DEBUG in a separate PR.